website/docs/docs/use-dbt-semantic-layer/exports.md

```diff
@@ -0,0 +1,179 @@
+---
+title: "Materialize queries with exports"
+description: "Use exports to materialize tables to the data platform on a schedule."
+sidebar_label: "Materialize with exports"
+---
+
+Exports with the dbt Semantic Layer enhance the [saved queries](/docs/build/saved-queries) feature, allowing you to materialize commonly used queries directly within your data platform.
+
+While saved queries are a way to save and reuse commonly used queries in MetricFlow, exports take this functionality a step further by:
+
+- Enabling you to materialize these queries within your data platform using dbt Cloud.
+- Proving an integration path for tools that don't natively support the dbt Semantic Layer by exposing tables of metrics and dimensions.
+
+Essentially, exports are like any other table in your data platform. They allow you to query metric definition through any SQL interface or connect to downstream tools without needing a first class Semantic Layer integration. Refer to [Available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) for more information.
```

Comment:
non-blocking sugg. i was taught to avoid anthropomorphism in user docs, "enable" is preferred to "allow"  

```suggestion
Essentially, exports are like any other table in your data platform. They enable you to query metric definition through any SQL interface or connect to downstream tools without needing a first class Semantic Layer integration. Refer to [Available integrations](/docs/use-dbt-semantic-layer/avail-sl-integrations) for more information.
```